pico_project.py

```diff
@@ -1214,11 +1263,11 @@ def DoEverything(parent, params):
 # Check we have everything we need to compile etc
 c = CheckPrerequisites()
 
-## TODO Do both warnings in the same error message so user does have to keep coming back to find still more to do
+# TODO Do both warnings in the same error message so user does have to keep coming back to find still more to do
 
 if c == None:
     m = 'Unable to find the `' + COMPILER_NAME + '` compiler\n'
-    m +='You will need to install an appropriate compiler to build a Raspberry Pi Pico project\n'
+    m += 'You will need to install an appropriate compiler to build a Raspberry Pi Pico project\n'
     m += 'See the Raspberry Pi Pico documentation for how to do this on your particular platform\n'
```

Comment:
I'll leave it up to you to decide whether it comes within the scope of this PR or not, but note that you _could_ also do this with a triple-quoted string e.g.
```python3
m = '''Unable to find the `{}` compiler
You will need to install an appropriate compiler to build a Raspberry Pi Pico project
See the Raspberry Pi Pico documentation for how to do this on your particular platform
'''.format(COMPILER_NAME)

Reply:
These strings could potentially be pulled out into a separate file, but yes probably best for another PR.